# HG changeset patch
# User Rik <rik@octave.org>
# Date 1388532675 28800
#      Tue Dec 31 15:31:15 2013 -0800
# Node ID 0490de623fa0a3db7249bf76825c9ee60a2b6ca0
# Parent  05a53cc522b75fdaf590b866554556a9dc317a3c
maint: Use Octave coding conventions in cset 1e9cb13faa7a.

* ls.m: Use Octave coding conventions in cset 1e9cb13faa7a.

diff --git a/scripts/miscellaneous/ls.m b/scripts/miscellaneous/ls.m
--- a/scripts/miscellaneous/ls.m
+++ b/scripts/miscellaneous/ls.m
@@ -56,25 +56,25 @@ function retval = ls (varargin)
 
   if (! iscellstr (varargin))
     error ("ls: all arguments must be character strings");
   endif
 
   if (nargin > 0)
     args = tilde_expand (varargin);
     is_dos = (ispc () && ! isunix ());
-    if is_dos
+    if (is_dos)
       optsep = "/";
     else
       optsep = "-";
     endif
     idx = ! strncmp (args, optsep, 1);
     ## Enclose paths, potentially having spaces, in double quotes:
-    args(idx) = strcat ("\"", args(idx), "\"");    
-    if is_dos
+    args(idx) = strcat ('"', args(idx), '"');    
+    if (is_dos)
       ## shell (cmd.exe) on MinGW uses '^' as escape character
       args = regexprep (args, '([^\w.*? -])', '^$1');
       ## Strip UNIX directory character which looks like an option to dir cmd.
       if (args{end}(end) == '/')
         args{end}(end) = "";
       endif
     else
       args = regexprep (args, '([^\w.*? -])', '\\$1');
